<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1030</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Просмотр Плана Секции</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="section_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>775</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>775</width>
      <height>540</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>775</width>
      <height>540</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="section_combobox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>775</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>775</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>775</width>
      <height>32</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="students_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Студенты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lodgers_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Проживание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rooms_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Комнаты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="section_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>200</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Секции</string>
    </property>
   </widget>
   <widget class="QPushButton" name="groups_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Группы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="building_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>300</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Корпусы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commandants_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Комменданты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="faculty_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Факультеты</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
